graphics/tinygl/init.cpp

```diff
@@ -261,7 +261,7 @@ void GLContext::init(int screenW, int screenH, Graphics::PixelFormat pixelFormat
 	// color mask
 	color_mask_red = color_mask_green = color_mask_blue = color_mask_alpha = true;
 
-	const int kDrawCallMemory = 5 * 1024 * 1024;
```

Comment:
that allocation is too big, it's 1GB RAM! try find matching value

Reply:
I had forgotten about this. Testing a complex scene it looks like 60mb is the maximum of what is needed, though it's usually less (around 20mb)